react/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FC } from 'react'
+
+const Header: FC = ({ children }) => {
```

Comment:
Does this component make sense? If so, shouldn't you add a handle to it?

Reply:
I need a block that accepts `children` for the header. Is there any other way to do that? Because its children go to a different place than the `index.tsx` children. But yeah, I'm going to add a handle to it!